packages/lesswrong/server/serverStartup.ts

```diff
@@ -31,6 +57,17 @@ async function serverStartup() {
   });
   
   const commandLineArguments = getCommandLineArguments();
+
+  await setMongoClient(commandLineArguments.mongoUrl);
+
+  onStartup(() => {
+    setInterval(async () => {
+      const memoryUsage = process.memoryUsage()?.heapTotal;
+      if (memoryUsage > consoleLogMemoryUsageThreshold.get() * 2) {
```

Comment:
Yikes. I don't love reusing a setting whose name tells me it's purely for logging for controlling somewhat risky behavior.